src/commands/send.rs

```diff
@@ -132,19 +132,19 @@ async fn send(message: &Ingress, opts: &SendOpts) -> AnyhowResult {
 }
 
 enum SupportedResponse {
-    ManageNeuronResponse,
-    TransferResponse,
-    AccountBalanceResponse,
+    ManageNeuron,
+    Transfer,
+    AccountBalance,
```

Comment:
I guess this is unrelated to the issues mentioned in the title. Also, I guess this does not affect any externally visible interfaces.

Reply:
Updated the title right after i sent out the PR. Refresh should have the updated scope. I had to change this to get cargo clippy and cargo fmt to pass